lua/ts_context_commentstring/config.lua

```diff
@@ -63,7 +63,9 @@ M.config = {
     c = { __default = '// %s', __multiline = '/* %s */' },
     css = '/* %s */',
     glimmer = '{{! %s }}',
+    go = '// %s',
```

Comment:
Since Go can have multiline comments with `/* */`, then I would also add those while we're at it:

```suggestion
    go = { __default = '// %s', __multiline = '/* %s */' },
```

Reply:
yeah I didn't know that it seems to be part of spec